packages/central-server/src/apiV2/GETEntityTypes.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { GETHandler } from './GETHandler';
+import { hasBESAdminAccess } from '../permissions';
+import { generateLinkHeader } from './GETHandler/helpers';
+
+const createEntityTypesDBFilter = async (accessPolicy, models, criteria) => {
+  if (hasBESAdminAccess(accessPolicy)) {
+    return criteria;
+  }
+
+  const dbConditions = { ...criteria };
+
+  return dbConditions;
+};
+
+export class GETEntityTypes extends GETHandler {
+  permissionsFilteredInternally = true;
+
+  async getPermissionsFilter(criteria, options) {
+    const dbConditions = await createEntityTypesDBFilter(this.accessPolicy, this.models, criteria);
+    return { dbConditions, dbOptions: options };
+  }
+
+  async buildResponse() {
+    let options = await this.getDbQueryOptions();
+    let criteria = this.getDbQueryCriteria();
+    if (this.permissionsFilteredInternally) {
+      ({ dbConditions: criteria, dbOptions: options } = await this.applyPermissionsFilter(
+        criteria,
+        options,
+      ));
+    }
+    const { types } = this.models.entity;
+    const entityTypes = Object.values(types)
```

Comment:
Technically this endpoint should return all types, including world. There may be another frontend that wants to list it.

You can then either just leave World in the dropdown and leave it to users to ignore it or add this same filter in admin-panel instead (not sure how off the top of my head but should be possible).

Reply:
This seems to still be the case, but it's minor